src/res_core.ml

```diff
@@ -1391,12 +1412,12 @@ and parseRecordPatternField p =
 and parseRecordPatternItem p =
   match p.Parser.token with
   | DotDotDot ->
-    Parser.next p;
-    Some (true, PatField (parseRecordPatternField p))
+    Parser.expect DotDotDot p;
+    Some (true, PatField (parseRecordPatternField ~label:(parseValuePath p) p))
```

Comment:
Here we *don't* know what the current token is.
But, termination analysis is happy so we're good.